runtime/src/serde_snapshot/tests.rs

```diff
@@ -719,7 +719,7 @@ mod test_bank_serialize {
 
     // This some what long test harness is required to freeze the ABI of
     // Bank's serialization due to versioned nature
-    #[frozen_abi(digest = "8aBQnNxnfKDxGEmMwaq9uLeMh3fw5sKsMGZBsCp15Dmv")]
+    #[frozen_abi(digest = "9BucA5MtPMNNUjADyV27vNgzvDy1RqCLH2gRq5NEuDEF")]
```

Comment:
this doesn't get saved until the feature is enabled, so not relevant that abi hash changes.